documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,287 @@
+
+
+.VMs from vSphere with RHEL 9 guest operating system can start with network interfaces that are down
+
+When migrating VMs that are installed with RHEL 9 as guest operating system from vSphere, the network interfaces of the VMs could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
+
+
+.Leftover `ovirtvolumepopulator` from failed migration causes the plan to stop in the `CopyDisks` phase
+
+An earlier failed migration can leave an outdated `ovirtvolumepopulator` in the project of a new plan for the same VM. The `CreateDataVolumes` phase does not create populator PVCs when transitioning to `CopyDisks`, causing the `CopyDisks` phase to stay indefinitely. link:https://issues.redhat.com/browse/MTV-929[(MTV-929)]
+
+
+.Unclear error message when Forklift fails to build or create a PVC
+
+The migration fails to build the PVC when the destination storage class does not have a configured storage profile. The error logs lack clear information to identify the reason for the failure. link:https://issues.redhat.com/browse/MTV-928[(MTV-928)]
+
+
+.Migration of a VM with NVME disks from vSphere fails
+
+When migrating a virtual machine (VM) with NVME disks from vSphere, the migration process fails, and the Web Console shows that the `Convert image to kubevirt` stage is `running` but did not finish successfully. link:https://issues.redhat.com/browse/MTV-963[(MTV-963)]
+
+
+.Importing image-based VMs can fail
+
+Migrating an image-based VM without the `virtual_size` field can fail on a block mode storage class. link:https://issues.redhat.com/browse/MTV-946[(MTV-946)]
+
+
+.Deleting migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs and data volumes. You must archive a migration plan before deleting it to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+
+.Migrating VMs with independent persistent disk from VMware to OCP-V fail
+
+Migrating Virtual Machines with independent persistent disk from VMware to OCP-V fail. link:https://issues.redhat.com/browse/MTV-993[(MTV-993)]
+
+
+.Migrating VM from one OCP-V cluster to another using timeouts and fails
+
+When trying to migrate VMs from one OCP-V cluster to another using {product-short}, the migration could timeout and fail. link:https://issues.redhat.com/browse/MTV-1052[(MTV-1052)]
+
+
+.Guest operating system from vSphere might be missing
+
+The guest operating system may be missing for VMs from vSphere. vSphere stops reporting the operating system after not receiving updates from VMware tools after some time. link:https://issues.redhat.com/browse/MTV-1046[(MTV-1046)]
+
+
+.{product-short} forklift-controller pod crashes in a loop after configuring vSphere provider
```

Comment:
@ahadas - please can you give us a bit more information about this ticket?
Thanks

Reply:
```suggestion
.{product-short} forklift-controller pod crashes when getting a disk without datastore
```
In some rare cases, the VM may be configured with a disk that is placed on a datastore that is no longer available in vSphere. MTV gets such disks without their datastore. Previously, forklift-controller crashed in this situation, rendering {project-short} unusable. Now, MTV presents a critical validation for VMs with such disks and forklift-controller no longer crashes.